src/main/kotlin/com/moshbit/katerbase/MongoHelper.kt

```diff
@@ -406,3 +406,12 @@ class AggregationPipeline {
     }
   }
 }
+
+class QueryStats(
+  val executionPipeline: List<String>,
```

Comment:
```suggestion
  val winningPlan: List<String>,
```
pls reuse mongodb wording when possible